cmd/minikube/cmd/root.go

```diff
@@ -145,9 +145,12 @@ func initConfig() {
 	configPath := constants.ConfigFile
 	viper.SetConfigFile(configPath)
 	viper.SetConfigType("json")
-	err := viper.ReadInConfig()
-	if err != nil {
-		glog.Warningf("Error reading config file at %s: %v", configPath, err)
+	if err := viper.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+			glog.Warningf("Config file not found; ignore error if desired")
```

Comment:
This should just log a debug message, and continue. No warning, no "ignore error if desired" (it is)

Reply:
ok

Reply:
PTAL